libkodimote/kodi.cpp

```diff
@@ -121,6 +121,8 @@ Kodi::Kodi(QObject *parent) :
 //                        XDebug::addAllowedArea(XDAREA_);
                     }
                 }
+            } else {
+                koDebug() << "Missing debug area argument after -d. Opssible options are discovery, player, library, files, playlist, networkdata and eventclient.";
```

Comment:
Can't this be written using qDebug? I had to look (more than) twice to see the general area always works (as it's hardcoded added above) and this line would thus actually be written to stdout/stderr instead of the FakeDevice.

(didn't test/run (yet))